discord_commands/drop_simulations.py

```diff
@@ -22,14 +22,17 @@ async def cmd_finish(ctx, name: str, trials = 1, extraArg1 = "True", extraArg2 =
         if name.lower() == "tob":
             if extraArg1 == "True":
                 extraArg1 = "4"
-            title = "Simulation of " + str(trials) + " Theatre of Blood completions, with a team size of " + extraArg1
+            if trials > 1:
```

Comment:
Possibly better to use the approach below? i.e "completion(s)" rather than adding another if/else branch. This entire command could do with refactoring, it's kind of messy.